pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>io.airlift</groupId>
         <artifactId>airbase</artifactId>
-        <version>168</version>
+        <version>170</version>
```

Comment:
>  Update google cloud SDK to 26.45.0

Looks like this commit doesn't change google cloud SDK? Are you going to squash into the 1st commit?

Reply:
Yes. First two commits come together

Reply:
Done.